src/extensions/default/BrambleUrlCodeHints/style.css

```diff
@@ -106,3 +106,956 @@
 #selfie-video-bg.on {
     opacity: 1;
 }
+
+#filter-list {
```

Comment:
I *love* that you tried to do this with pure CSS.  Unfortunately, it's not going to be possible to get the rendered version of the image into a canvas.  When we copy the image into the canvas, it's only able to access the image's data, not the CSS applied to it in the DOM.  It would be awesome if this worked, but it won't.

What we need is a JS library that can apply these filters to the canvas itself.  So you'd copy the image into the canvas, and use JS to apply the filter.